.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Question
+about: What's something you need help with?
+title: "PackageName:  General Summary of the Question"
+labels: "type: question"
+assignees: ''
+
+---
+
+## Question
```

Comment:
Not even sure if we need the header here.  The whole thing is a question.

Reply:
Yes, the though was that each template have a level two header indicating its type.  Seems that's more visible than the label off to the right, but I could be swayed.